src/physics/em/detail/MollerBhabhaInteractor.i.hh

```diff
@@ -43,8 +43,8 @@ CELER_FUNCTION MollerBhabhaInteractor::MollerBhabhaInteractor(
                  || particle.particle_id() == shared_.positron_id);
 
     secondary_energy_cutoff_ = celeritas::max(
-        secondary_energy_cutoff_,
-        (inc_particle_is_electron_ ? 2 : 1) * shared_.min_valid_energy());
+        (inc_particle_is_electron_ ? 0.5 : 1) * secondary_energy_cutoff_,
```

Comment:
Hate to be picky like this after all the back and forth, but this promotes to `double` (if `real_type` isn't double already), so maybe
```suggestion
        real_type(inc_particle_is_electron_ ? 0.5 : 1) * secondary_energy_cutoff_,
```